kratos/input_output/vtk_output.cpp

```diff
@@ -599,6 +631,26 @@ void VtkOutput::WriteVectorSolutionStepVariable(
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<typename TContainerType>
+void VtkOutput::WriteFlagContainerVariable(
+    const TContainerType& rContainer,
+    const Flags Flag,
+    const std::string& rFlagName,
+    std::ofstream& rFileStream) const
+{
+    rFileStream << rFlagName << " 1 "
+                << rContainer.size() << "  float\n";
+
+    for (const auto& r_entity : rContainer) {
+        const bool result = r_entity.Is(Flag);
```

Comment:
@adityaghantasala and I discussed this before and agreed on the following:

The value should be
-1 for undefined flags
0 for defined flags that are false
1 for defined flags that are true

Reply:
OK